<launch>
<node pkg="ins_positioning" type="i_positioning" name="i_positioning" output="screen"  >
  </node>
</launch>
